app/assets/stylesheets/pages/_signup.scss

```diff
@@ -0,0 +1,87 @@
+.signup-body {
+  background: linear-gradient(333deg, rgba(102,126,234,1) 0%, rgba(118,75,162,1) 100%);
+  padding: 10%;
+}
+
+.form-box {
+  background: #FAFAFA;
+  border-radius: 110px;
+  padding: 8%;
+}
+
+.signup-title {
+  text-align: center;
+  color: $main-purple;
+  border-bottom: 1px solid $main-purple;
+  padding-bottom: 5%;
+  margin-bottom: 5%;
+  h2 {
+    font-size: 4rem;
+  }
+}
+
+.availability {
+  // visibility: hidden;
+  display: none;
+}
+
+#availability-subtitle {
+  margin: 5% 0;
+  color: #764BA2;
+}
+
+.availabilityRow {
+  display: flex;
+  .box {
+    width: 50px;
+    text-align: center;
+  }
+}
+
+#row-1 {
+  border-bottom: 1px solid #764BA2;
```

Comment:
this color is repeated, let's extract it into a variable

Reply:
I just realised I had already made a bootstrap variable with it and forgot to use it~ 
I'll replace it and push again.